hax/hax/motr/__init__.py

```diff
@@ -20,7 +20,7 @@
 import logging
 from errno import EAGAIN
 from typing import Any, List
-from queue import Queue
+# from queue import Queue
```

Comment:
No need to comment the code out (unless this is a testing of a WIP branch, of course). Please remove the code you don't need.